test/unit/math/rev/err/check_bounded_test.cpp

```diff
@@ -1,7 +1,32 @@
+#include <stan/math/rev/arr.hpp>
 #include <stan/math/rev/scal.hpp>
 #include <gtest/gtest.h>
+#include <vector>
 #include <limits>
 
+TEST(AgradRevErrorHandlingScalar, CheckBoundedVarCheckVectorized) {
+  using stan::math::check_bounded;
```

Comment:
In other files the `using` statement for the function being tested was outside of the test blocks.

Reply:
I personally think that the using statements should be limited inside the tests as there is a possibility that global using statements could cause problems for the jumbo-builds.